ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly.yaml

```diff
@@ -275,6 +275,17 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: sdn-migration-ovn-aws-ipi
```

Comment:
@zhaozhanqi the name better be `e2e-aws-ipi-sdn-migration-ovn`

Reply:
ok, will fix this

Reply:
done